docs/website/versioned_docs/version-3.0.0/command-reference/dev.md

```diff
@@ -226,3 +226,23 @@ components:
     memoryLimit: 1024Mi
     mountSources: true
 ```
+
+### State file
+
+When the command `odo dev` is executed, the state of the command is saved in the file `./.odo/devstate.json`. 
+
+This state file contains the PID of the `odo dev` running process (`0` if no command is running), the timestamp 
```

Comment:
Whoops - I just noticed that this statement about storing the PID and the timestamp is no longer true. So this sentence needs to be updated accordingly.

Reply:
oh, my bad, removing it